test/1.0/1.0-tests.js

```diff
@@ -338,4 +338,128 @@ describe('RAML 1.0 Tests', function () {
       chai.expect(stdoutLines[2]).to.contain(files[2]);
     });
   });
+
+  describe('Invalid file with one warning (default)', function () {
+
+    const ramlCop = path.join(__dirname, '..', '..', 'src', 'raml-cop.js');
+    const file    = path.join(__dirname, 'data', 'invalid-one-warning.raml');
+    
+    let results = {};
+      
+    before(function (done) {
+
+      testChildProcess(ramlCop, [file], (err, data) => {
+        if (err) { return done(err); }
+
+        results = data;
+        done();
+      });
+    });
+
+    it ('STDOUT should contain filename, line number, column number, and message for both error and warning', function() {
```

Comment:
Description is wrong. There should be only one message and it should be a warning message.